src/imitation/util/video_wrapper.py

```diff
@@ -75,3 +79,69 @@ def close(self) -> None:
             self.video_recorder.close()
             self.video_recorder = None
         super().close()
+
+
+def record_and_save_video(
+    output_dir: str,
+    policy: policies.BasePolicy,
+    eval_venv: vec_env.VecEnv,
+    video_kwargs: Mapping[str, Any],
+    logger: Optional[sb_logger.Logger] = None,
+) -> None:
+    video_dir = osp.join(output_dir, "videos")
+    video_venv = VideoWrapper(
+        eval_venv,
+        directory=video_dir,
+        **video_kwargs,
+    )
+    sample_until = rollout.make_sample_until(min_timesteps=None, min_episodes=2)
```

Comment:
Why 2 episodes (if we only store a single video)?

Reply:
yeah, 1 episode should be enough